pkg/serverless/daemon/daemon.go

```diff
@@ -142,6 +143,11 @@ type Flush struct {
 // ServeHTTP - see type Flush comment.
 func (f *Flush) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	log.Debug("Hit on the serverless.Flush route.")
+	if len(os.Getenv("DD_LOCAL_TEST")) > 0 {
```

Comment:
Nit: I would make a constant for the env var (yeah, it's only used once so far but it's a good pattern to follow I think)